### PR TITLE
Add a vite.config.js file

### PR DESCRIPTION
**Pull request by CaptainCodex**
- Add the vite react plugin to the config
- Add yarn scripts commands to package.json `vite:dev` and `vite:build`

### CODE REVIEW

**Comment by CaptainCodex** [comment]
Add `vite-plugin-react` package. #

**Comment by CaptainCodex** [comment]
Add `vite-plugin-react` package.

**Comment by GrantDawsonAssistant** [comment]
Can we add `vite-plugin-react` package.

**Review by GrantDawsonAssistant — CHANGES_REQUESTED** [review]
Add `vite-plugin-react` package.

**Review by GrantDawsonAssistant — COMMENTED** [review]
Review comment

**Comment by GrantDawsonAssistant** [comment]
> Review comment

this reply

**Review by GrantDawsonAssistant — CHANGES_REQUESTED** [review]
revert file

**Review by GrantDawsonAssistant — COMMENTED** [review]
review comment 2

**Review by GrantDawsonAssistant — CHANGES_REQUESTED** [review]
this should be changed

File: package.json [hunk 1]
```diff
@@ -6,16 +6,7 @@
...
-  "license": "ISC",
   "dependencies": {
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "@chakra-ui/react": "^1.6.5",
-    "@emotion/react": "^11.4.1",
-    "@emotion/styled": "^11.3.0",
-    "framer-motion": "^4.1.17",
-    "vite": "^5.0.2"
+    "vite-plugin-react": "^1.0.0"
```
Comment (GrantDawsonAssistant) [inline package.json:10]: highlighted code

Comment (GrantDawsonAssistant) [reply package.json:10]: conversing

**Review by GrantDawsonAssistant — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -6,16 +6,7 @@
...
-  "author": "",
-  "license": "ISC",
   "dependencies": {
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "@chakra-ui/react": "^1.6.5",
-    "@emotion/react": "^11.4.1",
-    "@emotion/styled": "^11.3.0",
-    "framer-motion": "^4.1.17",
-    "vite": "^5.0.2"
```
Comment (CaptainCodex) [inline package.json:19]: hi

**Review by CaptainCodex — COMMENTED** [review]
lo

File: package.json [hunk 3]
```diff
@@ -6,16 +6,7 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
   },
```
Comment (GrantDawsonAssistant) [inline package.json:8]: pop

**Review by GrantDawsonAssistant — CHANGES_REQUESTED** [review]

**Comment by GrantDawsonAssistant** [comment]
> this should be changed (edit 3)

**Pull request closed without merging**